megamek/src/megamek/client/bot/princess/BasicPathRanker.java

```diff
@@ -58,6 +59,8 @@ public class BasicPathRanker extends PathRanker {
     private final NumberFormat LOG_INT = NumberFormat.getIntegerInstance();
     protected final NumberFormat LOG_PERCENT = NumberFormat.getPercentInstance();
 
+    private final
+
```

Comment:
I think this is the hangup.

Reply:
Yeah, I found that one here and fixed it.